grl/mdp.py

```diff
@@ -289,4 +287,4 @@ def test():
     print('All tests passed.')
 
 if __name__ == '__main__':
-    test()
```

Comment:
Is there a way to get yapf to take care of ensuring there's a newline at the end of every file?

Reply:
Maybe we can use this: https://github.com/editorconfig-checker/editorconfig-checker/

Reply:
Actually this is probably way easier:

https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3